api-server/config.py

```diff
@@ -6,3 +6,8 @@
 GITHUB_ACCESS_TOKEN = os.environ.get("GITHUB_ACCESS_TOKEN")
 DB_URI = os.environ.get("DB_URI")
 TEST_DB_URI = os.environ.get("TEST_DB_URI")
+GITHUB_CLIENT_ID = os.environ.get("GITHUB_CLIENT_ID")
```

Comment:
How do I generate `GITHUB_CLIENT_ID`, `GITHUB_CLIENT_SECRET` and etc...?

Reply:
You have to create an `OAuth Application` here https://github.com/settings/developers. 

Just make sure to set the following properties:
- Homepage URL to `http://127.0.0.1:8001`
- Authorization Callback URL to `http://127.0.0.1:8001/auth/gh`

Reply:
Got it!